indico/web/client/js/react/components/pictures/PictureCropper.jsx

```diff
@@ -0,0 +1,86 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Cropper from 'react-cropper';
+import {Header, Icon, Button} from 'semantic-ui-react';
+
+import 'cropperjs/dist/cropper.css';
+import './Picture.module.scss';
+
+export function PictureCropper({cropperRef, src, onCrop, backAction, minCropSize}) {
+  let size = Math.max(minCropSize, 500);
+  size += 200;
+
+  return (
+    <div styleName="picture-outer-div">
+      <div styleName="picture-inner-div cropper-inner-div" style={{width: size}}>
+        <Header as="h3" color="grey">
+          Crop
+        </Header>
+        <Cropper
+          ref={cropperRef}
+          style={{width: size, height: size}}
+          zoomTo={0}
+          initialAspectRatio={1}
+          src={src}
+          background={false}
+          minCropBoxHeight={minCropSize}
+          minCropBoxWidth={minCropSize}
+          autoCropArea={1}
+          checkOrientation={false}
+          guides
+          rotatable
+        />
+        <Icon
+          styleName="back"
+          name="arrow left"
+          color="grey"
+          size="large"
+          onClick={e => {
+            e.stopPropagation();
```

Comment:
```suggestion
          onClick={evt => {
            evt.stopPropagation();
```

more consistent with the rest of the codebase